
Trim INCLUDES in VPATH builds.

Upstream-HEAD: committed
Upstream-2.0: omit
Upstream-Status: Build system cleanup, not worth backport

--- httpd-2.0.48/configure.in.vpathinc
+++ httpd-2.0.48/configure.in
@@ -121,7 +121,7 @@
 
 if test "$abs_builddir" != "$abs_srcdir"; then
   USE_VPATH=1
-  APR_ADDTO(INCLUDES, [-I\$(srcdir) -I\$(top_builddir)/server/mpm/\$(MPM_SUBDIR_NAME) -I\$(top_builddir)/include])
+  APR_ADDTO(INCLUDES, [-I\$(top_builddir)/include])
 fi
 
 APR_ADDTO(INCLUDES, [-I\$(top_srcdir)/os/\$(OS_DIR) -I\$(top_srcdir)/server/mpm/\$(MPM_SUBDIR_NAME) -I\$(top_srcdir)/modules/http -I\$(top_srcdir)/modules/filters -I\$(top_srcdir)/modules/proxy -I\$(top_srcdir)/include])
